Client.xcodeproj/xcshareddata/xcschemes/Fennec_Enterprise_XCUITests.xcscheme

```diff
@@ -269,6 +269,18 @@
             isEnabled = "YES">
          </CommandLineArgument>
       </CommandLineArguments>
+      <EnvironmentVariables>
```

Comment:
This may need to be added to the Fennec_Enterpise_XCUITests_iPad schema too so that the test run successfully on iPad too (or disable the test for that schema)